packages/mask/src/plugins/CyberConnect/SNSAdaptor/Profile.tsx

```diff
@@ -0,0 +1,155 @@
+import { useEffect, useState } from 'react'
+import { makeStyles } from '@masknet/theme'
+import Avatar from 'boring-avatars'
```

Comment:
Please use [MUI avatar](https://mui.com/components/avatars/#main-content) instead.

Reply:
For UI performance, we decided to use boring-avatar here